gef-ui/templates/partials/confirm-cover-start-date.njk

```diff
@@ -35,13 +38,61 @@
         </div>
     </div>
 
-    <h1 data-cy="confirm-cover-start-date-heading">Confirm cover start date for issued facilities</h1>
+    {% if facility.length > 0 %}
+        {% if success %}
+            {{
+                mojBanner({
+                type: 'success',
+                text: success.message,
+                iconFallbackText: 'Success'
+                })
+            }}
+        {% endif %}
+    {% endif %}
 
-    {% if isUkefReviewAvailable %}
+    {% if isUkefReviewPositive %}
         <div class="govuk-grid-row govuk-body">
             <div class="govuk-grid-column-two-thirds">
-                <form method="POST" autocomplete="off">
-                </form>
+                {% if facility.length > 0 %}
+                    {{ govukTable
+                        ({
+                            caption: "Confirm cover start date for issued facilities",
+                            captionClasses: "govuk-table__caption--xl",
+                            firstCellIsHeader: true,
+                            classes: "govuk-table__vertical-align",
+                            head: [
+                                {
+                                text: "Faciliy name"
+                                },
+                                {
+                                text: "UKEF Facility ID"
+                                },
+                                {
+                                text: "Value"
+                                },
+                                {
+                                }
+                            ],
+                            rows: facility
+                        })
+                    }}
+                {% else %}
```

Comment:
is this condition enough - do we need another flag/property to use? Something like `allFacilityDatesConfirmed`?

I'm thinking if someone goes to this route/url directly, will they see this success message?

Maybe handle in the UI server, redirect to error page or main deal page. Question for design?

Reply:
This is enough as in UI server only the facilities which are not confirmed are returned by invoking filter()